go/chat/boxer.go

```diff
@@ -563,9 +575,11 @@ func (b *Boxer) ValidSenderKey(ctx context.Context, sender gregor1.UID, key []by
 	}
 
 	if deleted {
-		// XXX something else to flag as a deleted key?
-		b.log().Warning("sender %s key %s was deleted", kbSender, kid)
-		return true, true, nil, nil
+		b.Debug(ctx, "sender %s key %s was deleted", kbSender, kid)
+		// Set the key as being revoked since the beginning of time, so all messages will get labeled
+		// as suspect
+		zeroTime := gregor1.Time(0)
```

Comment:
This is a key change, we talked about it earlier, but I figured I'd point it out.